### ISSUE TITLE
Updating Windows installer for 0.17

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite-installers#424]
## Summary
- [ ] `compileymltojson` is no longer a command
- [ ] Docs are built into static-libraries/docs with the `make docs` target, this should be invoked or copied into the `bamboo-build.sh` script?
- [ ] Test that the `kalite.bat` executable still works.
- [ ] Use the new auto-discovered content pack (optional)
## Version
- Installer: Windows
- Version: 0.17b1

**Comment by benjaoming** [comment]
Btw. try to see if the Windows installer can bundle the .whl build instead of the tarball?

**Comment by mrpau-richard** [comment]
@benjaoming I think we can use the .whl build in `KA Lite` windows because We already did it in `Kolibri` windows installer.

**Comment by mrpau-eduard** [comment]
@benjaoming I try to run `make dist` command so that I can generate `.whl` file. I encountered an issue:
```
Build finished. The HTML pages are in _build/html.
cp -Rf docs/_build/html kalite/static-libraries/docs/
# open docs/_build/html/index.html
# Necessary because NPM may have wrong versions in the cache
npm cache clean
npm install --production
npm WARN package.json ka-lite@0.0.0 license should be a valid SPDX license expression
node build.js
Watchify: Found 20 bundles, compiling.
Watchify: Assuming that Khan Util files are already deAMDified
Watchify: 20 Bundles written.
KALITE_HOME=.kalite_dist_tmp bin/kalite manage syncdb --noinput
Traceback (most recent call last):
  File "bin/kalite", line 120, in <module>
    from kalite import cli
ImportError: No module named kalite
make: *** [assets] Error 1
(kalite_builds) ECHO-AIR:ka-lite mrpau-eduard$
```

**Comment by mrpau-eduard** [comment]
@benjaoming I get it now. I forgot to issue `pip install -e .` to install `kalite` in my virtualenv.
Its building now a `.whl` file.

**Comment by mrpau-richard** [comment]
fixed in #427
